test-cases/performance/perf-regression-write-cdc.yaml

```diff
@@ -0,0 +1,26 @@
+test_duration: 300
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -pop seq=1..30000000"
+n_db_nodes: 3
+n_loaders: 4
```

Comment:
why the perf-regression-mixed-cdc.yaml job only has one loader, but this job has 4 loaders?

Reply:
it is staging still

Reply:
will be moved to another PR